platform/crd-ctrlr/src/controllers/mwan3policy_controller.go

```diff
@@ -17,19 +17,110 @@ package controllers
 
 import (
 	"context"
-	"time"
-
+	// "time"
+	"encoding/json"
+	"fmt"
 	"github.com/go-logr/logr"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	// "k8s.io/apimachinery/pkg/api/errors"
+	// metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Instead of commenting out these lines, let's remove them directly. As well as the other comments